spec/models/concerns/requests/scsb_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe Requests::Scsb do
+  let(:user) { FactoryBot.build(:user) }
+  let(:valid_patron) do
+    { "netid" => "foo", "first_name" => "Foo", "last_name" => "Request",
+      "barcode" => "22101007797777", "university_id" => "9999999", "patron_group" => "staff",
+      "patron_id" => "99999", "active_email" => "foo@princeton.edu" }.with_indifferent_access
+  end
+  let(:patron) do
+    Requests::Patron.new(user: user, session: {}, patron: valid_patron)
+  end
+  let(:scsb_no_format) { fixture('/SCSB-7935196.json') }
+  let(:location_code) { 'scsbnypl' }
+  let(:params) do
+    {
+      system_id: 'SCSB-7935196',
+      source: 'pulsearch',
+      mfhd: nil,
+      patron: patron
+    }
+  end
+  let(:request_scsb) { Requests::Request.new(params) }
+  let(:first_item) { request_scsb.holdings["8076325"]["items"][0] }
+  let(:second_item) { request_scsb.holdings["8076325"]["items"][1] }
+
+  context 'with a good scsb key', vcr: { cassette_name: 'good_ol_good_bibdata_scsb_key', record: :none } do
```

Comment:
@maxkadel thank you.
Instead of bad and good can we use authorized and unauthorized?

Reply:
Thanks for looking it over - I've changed to authorized / unauthorized.